### DISCUSSION TITLE
Increasing snapserver buffer in Runeaudio

### DISCUSSION THREAD

**Discussion by skycamel** [snapcast/snapcast#859] [Q&A]
Hi,
I'm experimenting with Runeaudio and snapcast.
Runeaudio is running on a RPI 3B and I have a snapcast client running on a Android smartphone.
I'm listening to my web radio on my wireless wearable speaker connected over bluetooth to my Runeaudio
and simultaneously streaming to my snapcast client on an Android phone (connected to a hifi)
The problem is that the sound to my bluetooth is a lot "faster" than the streaming to my snapcast client. (yeah, i would have expected the other way around)
I have to set the sync delay on the snapcast client to over +800 ms so I have a lot of dropouts (I guess buffer 1000 ms?)
So is there a way in some config file of snapcast server to increase the buffer to say 1500 or 2000 ms?
I'm using the bluetooth wearable speakers because it's easier to understand the talking on the web radio but at the same time my wife wants to listen to the same station through the normal hifi at lower volume, hence snapcast perfect solution.
Any help much appreciated
Luc

**Comment by palitu** [comment]
Sorry @skycamel, quite off topic!  I am looking for the best way to easily install snapcast (client) as an appliance, and am trying to figure out whether  mopidy, volumio, rune etc are the way to go.  how'd you go about installing with Rune?  any links/ comments would be appreciated.

cheers!

**Reply by skycamel** [reply]
@palitu 
No worries. I only used RuneAudio so far and it's very straight forward to use snapcast as a server or as a client, both are possible. It's easily activated in the settings through the GUI, that's it.
Be aware that the last official release of  RuneAudio is "old" and there's a new version (that I'm using) maintained by a enthusiast @rern and is working great. You can find it on github as rAudio-1 in the rern repository.
It's also discussed in the RuneAudio forum.
Cheers,
Luc

**Reply by palitu** [reply]
@skycamel , works a treat, however i cannot find where to set the default input to be snapcast.  Do you know how to set that?

cheers

**Comment by palitu** [comment]
That. Is. So. Exciting!

Cheers man, i have a few pi's and amp hats just waiting for me to add something onto them :)

The wireless AP will really make the addition of new nodes that much easier, and hopefully the snapclient names can be set through the Raudio front end.

cheers
